src/cdk/testing/protractor/protractor-harness-environment.ts

```diff
@@ -37,6 +37,13 @@ export class ProtractorHarnessEnvironment extends HarnessEnvironment<ElementFind
     return new ProtractorHarnessEnvironment(protractorElement(by.css('body')), options);
   }
 
+  static getNativeElement(el: TestElement) {
```

Comment:
Since this method is not inheriting any JSDoc, should we add a description? Also it might be good to add an explicit return type. That is usually better than TS inferring it through various indirections (which is slow; and prone to signature changes)